wps-cloudformation-template.yaml

```diff
@@ -748,7 +748,7 @@ Resources:
       VpcConfig: !If
         - UseVpcGateway
         -
-          SecurityGroupIds: !Ref LambdaSecurityGroup
+          SecurityGroupIds: [!Ref LambdaSecurityGroup]
```

Comment:
The following format is more in line with the way arrays are being set in the rest of the template:
```
SecurityGroupIds:
    - !Ref LambdaSecurityGroup
```

Reply:
Modified so we can deploy to systest.  Might be worth establishing some coding conventions separately.  I prefer the abbreviated style when there is a small list which fits on the line.

Reply:
Yeah it's a bit mixed for this kind of reason... when I did a big style overhaul on this template I kind of went with the gut feeling about what style was suitable for what type of line, but tried to make it consistent.

Most of the short intrinsic function calls are one liners because it "felt" right in this kind of scenario, because the longhand looks like crap, e.g.:

```yaml
!FindInMap [Constants, ConstantMap, wpsApiStage]
```

```yaml
Fn::FindInMap:
  - Constants
  - ConstantMap
  - wpsApiStage
```

but other things look a bit cluttered with unnecessary special characters when they are stuffed on one line. For example with an extreme contrived example:

```
- PolicyName: S3Access
            PolicyDocument:
              Version: 2012-10-17
              Statement:
                - Effect: Allow
                  Action:
                    - s3:Delete*
                    - s3:List*
                    - s3:Get*
                  Resource:
                    - !Sub 'arn:aws:s3:::${EphemeralS3Bucket}'
                    - !Sub 'arn:aws:s3:::${EphemeralS3Bucket}/*'
```
```
- PolicyName: S3Access
            PolicyDocument:
              Version: 2012-10-17
              Statement: [{Effect: Allow, Action: [s3:Delete*, s3:List*, s3:Get*], Resource:[!Sub 'arn:aws:s3:::${EphemeralS3Bucket}', !Sub 'arn:aws:s3:::${EphemeralS3Bucket}/*']}]
```
I think there is basically no value in trying to reduce lines as an end goal, so whatever consistent convention makes for the best readability is my vote! Sometimes that is the short form to my eyes, and sometimes the long form.